synapse/lib/cli.py

```diff
@@ -351,8 +351,7 @@ def runCmdLoop(self):
                 if not line:
                     continue
 
-                task = s_glob.plex.coroToTask(self.runCmdLine(line))
-                task.result()
+                s_glob.sync(self.runCmdLine(line))
```

Comment:
So the previous code here got a task, and would result in cancelling it on a keyboard interrupt.  That code is no longer triggerable with this change.